backend/app/server/database.py

```diff
@@ -3,7 +3,7 @@
 
 config = dotenv_values(find_dotenv())
 
-MONGO_DETAILS = config["MONGODB_URL"]
+MONGO_DETAILS = "mongodb://localhost:27017"
```

Comment:
You have pushed the wrong file.. 👀 @aitikgupta

Reply:
Oh noooo xD
Let me add it back.

Reply:
Fixed 👍🏼